stanford_field_formatters.module

```diff
@@ -643,7 +643,7 @@ function _stanford_field_formatters_fontawesome_icon_widget_form(&$form, &$form_
         "$path/js/stanford_field_formatters.js",
       ),
     ),
-    '#default_value' => $items[$delta]['icon'],
```

Comment:
This wont work for a couple reasons. the default value has to be the key of the array options. `- None -` is not a key. Also something like this would make all default values none. even on the edit page.